python/pyspark/mllib/feature.py

```diff
@@ -447,6 +454,17 @@ class Word2Vec(object):
     >>> syms = model.findSynonyms(vec, 2)
     >>> [s[0] for s in syms]
     [u'b', u'c']
+
+    >>> import os, tempfile
+    >>> path = tempfile.mkdtemp()
+    >>> model.save(sc, path)
+    >>> sameModel = Word2VecModel.load(sc, path)
+    >>> model.transform("a") == sameModel.transform("a")
+    True
+    >>> try:
+    ...     os.removedirs(path)
```

Comment:
I understand. You're right. I have just read the specification of `os.removedirs()`. `shutil.rmtree()` would be better. Thanks!